assets/wire/reducers.js

```diff
@@ -114,19 +113,16 @@ function _wireReducer(state, action) {
     switch (action.type) {
 
     case TOGGLE_NAVIGATION: {
-        const activeProducts = {};
         const activeNavigation = {};
 
-        if (action.navigation && action.navigation.products) {
-            action.navigation.products.map((product) => activeProducts[product] = true);
+        if (action.navigation) {
             activeNavigation[action.navigation._id] = true;
```

Comment:
would still make this `activeNavigation = action.navigation && action.navigation._id;` there is no need for this to be a dict